src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerResolverTest.php

```diff
@@ -24,6 +24,24 @@
 
 class ControllerResolverTest extends ContainerControllerResolverTest
 {
+    public function testGetControllerWithServiceWithoutMethod()
+    {
+        class_exists(AbstractControllerTest::class);
+
+        $dummyController = new DummyController();
+        $container = new Container();
+        $container->set('app.index_controller', $dummyController);
+
+        $resolver = $this->createControllerResolver(null, $container);
+        $request = Request::create('/');
+        $request->attributes->set('_controller', 'app.index_controller');
+
+        $controller = $resolver->getController($request);
+
+        $this->assertSame($container, $controller[0]->getContainer());
+        $this->assertSame('__invoke', $controller[1]);
```

Comment:
@rjwebdev I would follow the same logic as in `testGetControllerOnContainerAwareInvokable`:

```php
$this->assertInstanceOf('Symfony\Bundle\FrameworkBundle\Tests\Controller\DummyController', $controller);
$this->assertInstanceOf('Symfony\Component\DependencyInjection\ContainerInterface', $controller->getContainer());
```